components/login/provider/index.jsx

```diff
@@ -46,6 +50,10 @@ const useStyles = makeStyles((theme) => createStyles(styles(theme)));
 const providers = getIdentityProviders();
 export const TESTCAFE_ID_LOGIN_FIELD = "login-field";
 const TESTCAFE_ID_GO_BUTTON = "go-button";
+const hostname = getCurrentHostname();
+const CLIENT_APP_WEBID = hostname?.includes("localhost")
```

Comment:
`getCurrentHostname` should be refactored to return only the hostname and not the origin; and this should do an equality check, rather than "includes" (otherwise this would match `mylocalhost.com`, and other weird urls.)

Reply:
This logic also seems to be re-used at least three times. Does a `isLocalhost` function make sense?

Reply:
added `getCurrentOrigin` as it was needed in order to build the `CLIENT_APP_WEBID` uri in the components, and edited `getCurrentHostname` to return just hostname. 

In stringHelpers added `isLocalhost(hostname)` which will return a `boolean` value based on an equality check.